SupersonicSound/Studio/System.cs

```diff
@@ -68,9 +68,11 @@ private void Dispose(bool disposing)
             {
                 if (disposing)
                 {
-                }
+                    LowLevelSystem.Dispose();
+                    LowLevelSystem = null;
 
-                _system.release().Check();
+                    _system.release().Check();
```

Comment:
Yea I think you're right. sub-objects should be inside the if statement, unmanaged resources (fmod) should be outside, at least what I can read from this (long) article: http://joeduffyblog.com/2005/04/08/dg-update-dispose-finalization-and-resource-management/
I'll update it.